src/modules/faqs/faqs.controller.ts

```diff
@@ -0,0 +1,15 @@
+import { Body, Controller, Post, Req, UseGuards } from '@nestjs/common';
+import { CreateFaqDto } from './dto/createFaqsDto';
+import { FaqsService } from './faqs.service';
+
+@Controller('admin/faqs')
+@UseGuards()
+export class FaqsController {
+  constructor(private readonly faqService: FaqsService) {}
+  @Post('/')
+  async handleCreateFaqs(@Req() req, @Body() createFaqs: CreateFaqDto) {
+    const user = req['user'];
```

Comment:
Document your route and remove the @UseGuards decorator

Reply:
I will do that now, thanks